ledger/ledger.go

```diff
@@ -454,6 +454,11 @@ func (l *Ledger) notifyCommit(r basics.Round) basics.Round {
 	}()
 	minToSave := l.trackers.committedUpTo(r)
 
+	// Check if additional block history is configured, and adjust minToSave if so.
+	if configuredMinToSave := r.SubSaturate(basics.Round(l.cfg.MaxBlockHistoryLookback)); configuredMinToSave < minToSave {
```

Comment:
Condition looks to be backwards. Presumably this value already takes into account the `1321` and `MaxAcctLookback` settings, so we need to ensure it doesn't shrink.
```suggestion
	if configuredMinToSave := r.SubSaturate(basics.Round(l.cfg.MaxBlockHistoryLookback)); configuredMinToSave > minToSave {
```

Reply:
I mistook the variable to be "min number of rounds to save" when it is actually "minimum round to save". Looks like this code is correct.